javalib/src/main/scala/java/lang/Integer.scala

```diff
@@ -105,20 +157,156 @@ object Integer {
     (((t2 + (t2 >> 4)) & 0xF0F0F0F) * 0x1010101) >> 24
   }
 
-  @inline def divideUnsigned(dividend: Int, divisor: Int): Int =
+  @inline def compare(x: scala.Int, y: scala.Int): scala.Int =
+    if (x == y) 0 else if (x < y) -1 else 1
+
+  @inline def compareUnsigned(x: scala.Int, y: scala.Int): scala.Int =
+    undefined
```

Comment:
A possible implementation is `compare(x ^ scala.Int.MinValue, y ^ scala.Int.MinValue)`.